department-of-veterans-affairs/va.gov-cms
# VACMS-4137: Open-CLP-Field-Groups-With-Required-Empty-Field

… field group and open field groups with required empty fields

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/4137

## Testing done


## Screenshots


## QA steps

- Login to preview env.
- Navigate here: /node/15068/edit
- Leave audience field empty and keep the Why this matters fieldset closed
- Click Save
- [x] Confirm that the "Why this matters" field group is now open and field validation triggered.
- [x] Repeat the same steps as above but with different fields, open a field group, delete the value in the field, close the field group and select save.

Expected behavior
Form validation behavior should work as expected.
### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
